bentoml/cli/bento_service.py

```diff
@@ -345,18 +340,15 @@ def install_completion(append, shell, path):
         '--build-arg', multiple=True, help="pass through docker image build arguments"
     )
     @click.option(
-        '-u', '--username', type=click.STRING, required=False,
-    )
-    @click.option(
-        '-p', '--password', type=click.STRING, required=False,
+        '--repository', type=click.STRING, required=False,
     )
     @click.option(
         '--yatai-url',
         type=click.STRING,
         help='Remote YataiService URL. Optional. '
```

Comment:
`help='Specify the YataiService for running the containerization, default to the Local YataiService with local docker deamon'`

Reply:
Could you update the inline docstring below, some of them are outdated